content/manage-repos-pop.md

```diff
@@ -17,70 +24,62 @@ section: software
 
-```bash
-flatpak remotes
-```
+![Repoman Extra Sources window](/images/manage-repos/Repoman-extra-sources.png)
 
-This is used to list the remote/sources configured on your system.
+There will be three buttons on the lower left. The **Plus** button is for adding new repositories by hand, although third-party software will usually do this automatically. The **Pencil** button will open up a selected repository for editing, including the ability to disable it without completely removing it.
 
-```bash
-flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
-```
+![Repoman repository edit window](/images/manage-repos/Repoman-edit-repo.png)
 
-This command is used to add a remote/source if it doesn't exist on your system and in this example the Flathub remove/source is being added.
+The **Trash** button will delete a selected software repository completely.
 
-```bash
-flatpak remote-delete flathub
-```
+## Repoman and Flatpak Sources
 
-If we want to remove a remote/source the following command can be used.
+<u>Repoman</u> can be also be used to add, remove, and edit Flatpak sources starting with Pop!_OS 20.04 LTS. Click on the **Flatpak** tab to view them, and there will be similar buttons for adding, editing, or removing sources.
 
-```bash
-flatpak search inkscape
-```
+![Repoman Flatpak tab](/images/manage-repos/Repoman-Flatpak-tab.png)
 
-This command is used to search for a certain package (inkscape in this case) in the coufigured remotes/sources on this system.
+Editing a Flatpak source will allow you to disable it without removing it, can also show you what software has been installed from that source.
 
-```bash
-flatpak install flathub org.inkscape.Inkscape
-```
+![Repoman Flatpak edit window](/images/manage-repos/Repoman-Flatpak-edit.png)
+
+## Repositories, Sources, and the Command Line
 
-Here we are installing inkscape using the Flathub remote/source.
+The <u>Repoman</u> tool can also be run from the command line:
 
 ```bash
-flatpak uninstall org.inkscape.Inkscape
+repoman
 ```
 
-This will remove the inkscape package from this system.
+There may also be situations where repositories and Flatpak sources need to be adjusted from the terminal. Editing system repositories from the command line is not recommended without the aid of technical support, as this can lead to a system that will not receive updates correctly, but Flatpak sources are normally safe to edit from the terminal.
+
+To list the currently configured Flatpak sources:
 
 ```bash
-flatpak run org.inkscape.Inkscape
+flatpak remotes
 ```
 
-This command will run inkscape once it is installed on the system.
+To add a new Flatpak source (with Flathub as the example):
 
 ```bash
-flatpak update
+flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
```

Comment:
This should have `--user` to add a flathub remote source as a user repository.

Reply:
That's how it was on the original document. I've gone ahead and added `--user` to both the `remote-add` and `remote-delete` commands for consistency.